mlflow/utils/autologging_utils.py

```diff
@@ -113,3 +118,65 @@ def wrap_patch(destination, name, patch, settings=None):
     wrapped = functools.wraps(original)(patch)
     patch = gorilla.Patch(destination, name, wrapped, settings=settings)
     gorilla.apply(patch)
+
+
+class _InputExampleInfo:
+    def __init__(self, input_example=None, error_msg=None):
+        self.input_example = input_example
+        self.error_msg = error_msg
+
+
+def handle_input_example_and_signature(
```

Comment:
Nit: Is there a better / more descriptive verb we can use than `handle`? `resolve`?

Reply:
i like resolve :)